lib/broccoli/translation-reducer.js

```diff
@@ -233,22 +242,25 @@ class TranslationReducer extends CachingWriter {
   build() {
     let outputPath = `${this.outputPath}/${this.options.outputPath}`;
     let translations = this.readDirectory(this.inputPaths[0], this.listFiles());
-    let translation;
-    mkdirp.sync(outputPath);
+    let fallbacks = translations[this.options.fallbackLocale];
 
-    this.handleLintResult(this.lintTranslations(translations));
+    mkdirp.sync(outputPath);
 
     for (let locale in translations) {
-      if (translations.hasOwnProperty(locale)) {
-        translation = translations[locale];
-
-        if (this.options.verbose) {
-          this.validateMessages(translation, locale);
-        }
+      if (this.options.verbose) {
+        this.validateMessages(translations[locale], locale);
+      }
 
-        fs.writeFileSync(`${outputPath}/${this.filename(locale)}`, this.wrapEntry(translation), { encoding: 'utf8' });
+      if (fallbacks && this.options.fallbackLocale !== locale) {
+        translations[locale] = extend(true, {}, fallbacks, translations[locale]);
       }
+
+      fs.writeFileSync(`${outputPath}/${this.filename(locale)}`, this.wrapEntry(translations[locale]), {
+        encoding: 'utf8'
+      });
     }
+
+    this.handleLintResult(this.lintTranslations(translations));
```

Comment:
I would still prefer to get the warnings, but they should be adjustable in some way so that they don't warn about translations that are indeed shared between locales

Reply:
Okay, I can move this logic so that then on fallback, you'll always be warned.  You'll then need to implement your custom [`requiresTranslation`](https://github.com/ember-intl/ember-intl/blob/master/docs/missing-translations.md#requiring-translations) to define which should be ignored.

Reply:
yep, that seems good, thanks!